thirdeye-server/src/main/java/ai/startree/thirdeye/core/AppAnalyticsService.java

```diff
@@ -79,4 +100,32 @@ private MonitoredMetricWrapper wrapMonitoredMetric(final AlertMetadataDTO metada
         .setDataset(metadata.getDataset().getName())
         .setDatasource(metadata.getDatasource().getName());
   }
+
+  public ConfusionMatrix computeConfusionMatrixForAnomalies() {
+    int truePositive = 0;
+    int falsePositive = 0;
+    int unclassified;
+    List<MergedAnomalyResultDTO> anomalies = anomalyManager.findAll();
+    List<AnomalyFeedbackType> feedbackTypes = anomalies.stream()
+        .filter(anomaly -> anomaly.getAnomalyFeedbackId() != null)
+        .map(anomaly -> anomalyManager.convertMergedAnomalyBean2DTO(anomaly,  new HashSet<>()).getFeedback().getFeedbackType())
+        .collect(Collectors.toList());
+    unclassified = anomalies.size() - feedbackTypes.size();
+    for (AnomalyFeedbackType type : feedbackTypes) {
+      switch (type) {
+        case NO_FEEDBACK:
+          unclassified++;
+          break;
+        case NOT_ANOMALY:
+          falsePositive++;
+          break;
+        default:
```

Comment:
Makes sense. Added separate handling of FeedbackTypes.
Not throwing an exception for invalid type, just logging an error for it as it may break the metric calculation.